### PR TITLE
created Hancitor signature

### PR DESCRIPTION
**Pull request by ronbarrey**
Signature for detecting Hancitor URLs and decrypting C2 traffic.

### CODE REVIEW

File: modules/signatures/windows/loader_hancitor.py [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    severity = 3
+    categories = ["loader"]
+    families = ["hancitor"]
+    authors = ["ronbarrey"]
+    minimum = "2.0"
+
+    urls_re = [
+        ".*api.ipify.org",
+        ".*com\/ls[0-9]\/.*.php",
+        ".*ru\/ls[0-9]\/.*.php",
```
Comment (doomedraven) [inline modules/signatures/windows/loader_hancitor.py]: probably better remove tlds as it can be changed easy as path, but it will be better in my opinion

**Comment by doomedraven** [comment]
jsut checked with one of the last chanitors, it not getting any real cnc, it detects only ipify, as cncs are in buffer, but not buffers which you checking, going to try fix it here and will provide update

**Comment by doomedraven** [comment]
it probably related to version oh hancitor, but i would suggest to you add check for

```
        buffers = self.get_results("buffer", [])
        for buffer in buffers:
            urls = buffer.get("urls", [])
            if urls:
                for url in urls:
                   #apply your regexes here too
```

**Comment by ronbarrey** [comment]
What was the sample you checked against? Normally, I see the URLs in the process memory.

**Comment by doomedraven** [comment]
any with /ls5/ which comes from docs with process hollowing, /ls6/ change and not works with any of this signature

**Comment by ronbarrey** [comment]
Updated the signature per recommendations. Need to test against other Hancitor versions. @doomedraven do you have a hash for ls6?

**Comment by doomedraven** [comment]
https://www.virustotal.com/es/file/4c0f7f9ab747e3e640a374f97edbe7ff93c87702874d0cb8d9922db1e2ee0670/analysis/

**Comment by jbremer** [comment]
As this is a pure network traffic-based Signature, would it make sense to port this logic to matching the extracted output from the `.pcap` / network capture file? That will not get you the URL if the VM has no internet access, but other than that should be fine.

**Comment by doomedraven** [comment]
if you hook correct apis, you can get it even with drop network traffic rule

**Pull request closed without merging**